pkgs/top-level/all-packages.nix

```diff
@@ -37738,6 +37738,8 @@ with pkgs;
 
   EmptyEpsilon = callPackage ../games/empty-epsilon { };
 
+  en-croissant = callPackage ../by-name/en/en-croissant/package.nix { };
+
```

Comment:
AFAIK, you won't need this when using `by-name`

Reply:
correct, this is not needed